natural_language_processing/extractive_question_answering/bert_large/run_huggingface.py

```diff
@@ -15,7 +15,7 @@
 def parse_args():
     parser = argparse.ArgumentParser(description="Run model from Huggingface's transformers repo for extractive question answering task.")
     parser.add_argument("-m", "--model_name",
-                        type=str, choices=["bert-large-uncased-whole-word-masking-finetuned-squad"], required=True,
```

Comment:
I do want this scripts to be dedicated to given model architectures, therefore the choices use. If this is about roberta etc. could you just create separate runners with own directories for those models? I want to keep this in-line with what we do with CV nets although there will be some code repetitions.